### PR TITLE
feat(camunda-platform): add user assignment group

### LINKED ISSUE #43: Implement user task props
image

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Closes #43 

Bildschirmfoto 2021-07-27 um 11 20 23

### CODE REVIEW

File: src/provider/camunda-platform/properties/UserAssignmentProps.js [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+      }
+    });
+  };
+
+  return TextField({
+    element,
+    id: 'followUpDate',
+    label: translate('Follow Up Date'),
+    description : translate('The follow up date as an EL expression (e.g. ${someDate} or an ' +
+      'ISO date (e.g. 2015-06-26T09:54:00)'),
```
Comment (barmac) [inline src/provider/camunda-platform/properties/UserAssignmentProps.js:219]:
```suggestion
    description : translate('The follow up date as an EL expression (e.g. ${someDate}) or an ' +
      'ISO date (e.g. 2015-06-26T09:54:00).'),
```

File: src/provider/camunda-platform/properties/UserAssignmentProps.js [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+        'camunda:dueDate': value
+      }
+    });
+  };
+
+  return TextField({
+    element,
+    id: 'dueDate',
+    label: translate('Due Date'),
+    description : translate('The due date as an EL expression (e.g. ${someDate} or an ISO date (e.g. 2015-06-26T09:54:00)'),
```
Comment (barmac) [inline src/provider/camunda-platform/properties/UserAssignmentProps.js:184]:
```suggestion
    description : translate('The due date as an EL expression (e.g. ${someDate}) or an ISO date (e.g. 2015-06-26T09:54:00).'),
```

**Review by barmac — APPROVED** [review]
LGTM

**Review by andreasgeier — APPROVED** [review]
✅ &nbsp; LGTM

**Pull request merged**